ci_framework/playbooks/99-logs.yml

```diff
@@ -31,6 +31,12 @@
     - name: Generate artifacts
       ansible.builtin.import_role:
         name: artifacts
+      when: not cifmw_os_must_gather|bool | default('false')
```

Comment:
IMHO one may not exclude the other - let's have a discussion with the must_gather folks and see where we do have overlap, where we don't, and how to properly converge.